core/src/components/radio-group/radio-group.tsx

```diff
@@ -21,6 +21,14 @@ export class RadioGroup implements ComponentInterface {
    */
   @Prop() allowEmptySelection = false;
 
+  /**
+   * This property allows developers to specify a custom function or property
+   * name for comparing objects when determining the selected option in the
+   * ion-radio-group. When not specified, the default behavior will use object
+   * equality (===) for comparison.
```

Comment:
```suggestion
   * ion-radio-group. When not specified, the default behavior will use strict
   * equality (===) for comparison.
```

I think this is actually called "strict equality" according to https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Strict_equality